.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+# This github action automatically publishes a release when we merge to the main branch.
+# THIS IS CURRENTLY A POC. It does not yet actually publish a release, but merely performs a dry run.
+name: auto-release
+on:
+  push:
+    branches:
+      - main
```

Comment:
**For tomorrow's meeting:**
We have the option to either let the Github action determine if a new release should happen (dangerous?) or we keep the decision within the team by merging develop branch and into main which will trigger a full release automatically.

Reply:
Yep, that's one of the items on my list 👍 :-)

Reply:
Outcome: For now we want to keep a manual trigger for actually publishing a release. We still need to decide if we want to realize that via two branches (development vs. main) or by manually triggering a job to publish a release. IMO we can leave that undecided for now, as this PR (and the GH action) will not actually release anything.